utils/uniset/README.md

```diff
@@ -7,37 +7,37 @@ This API provides necessary functionality for highly efficient querying of sets
 It is an implementation of the existing [ICU4C UnicodeSet API](https://unicode-org.github.io/icu-docs/apidoc/released/icu4c/classicu_1_1UnicodeSet.html).
 
 ## Architecture
-ICU4X [`CodePointSet`] is split up into independent levels, with [`CodePointSet`] representing the membership/query API,
-and [`CodePointSetBuilder`] representing the builder API.
+ICU4X [`CodePointInversionList`] is split up into independent levels, with [`CodePointInversionList`] representing the membership/query API,
+and [`CodePointInversionListBuilder`] representing the builder API.
 
 ## Examples:
 
-### Creating a `CodePointSet`
+### Creating a `CodePointInversionList`
 
-CodePointSets are created from either serialized [`CodePointSets`](CodePointSet),
+CodePointSets are created from either serialized [`CodePointSets`](CodePointInversionList),
 represented by [inversion lists](http://userguide.icu-project.org/strings/properties),
-the [`CodePointSetBuilder`], or from the TBA Properties API.
+the [`CodePointInversionListBuilder`], or from the TBA Properties API.
```

Comment:
Nit: The properties API is no longer "TBA"; fix?

Reply:
Done.